### DISCUSSION TITLE
如何更改评论数量的 Waline: 字样

### DISCUSSION THREAD

**Discussion by zhangnew** [walinejs/waline#1060] [Q&A] [answered]
想把这个 `Waline`修改为 `评论数`，代码中看起来有配置，但是不知道如何使用？
这段代码中的 `post.comments.${key}` 如何配置？hexo 的内置变量中 post.comments 是一个布尔类型
https://github.com/walinejs/waline/blob/b57612962fd462e61ef1461fe9ae79de7d1de8aa/packages/hexo-next/index.js#L17-L27

**Comment by lizheming** [comment, marked as answer]
https://github.com/next-theme/hexo-theme-next/blob/master/languages/zh-CN.yml 在 NexT 主题里定义一下 `post.comments.waline` 的值应该就行

**Reply by zhangnew** [reply]
感谢！确实可以，但是修改主题源码不太优雅，在 Next 文档中找到[另一种覆写方法](https://theme-next.js.org/docs/theme-settings/internationalization#Override-Default-Translations)测试可行，也就是创建 `source/_data/languages.yml` 文件，内容为：
```yaml
zh-CN:
  post:
    comments:
      waline: 评论数
```
